openshift/api.dc.yml

```diff
@@ -51,12 +51,12 @@ objects:
             valueFrom:
               secretKeyRef:
                 key: met-username
-                name: met-database-users
```

Comment:
can you revert this? for both demo and test we use met-database-users, I'll keep dev updated with that too!

Reply:
sure..yeah.i wasnt sure of this change